pkg/kubelet/eviction/eviction_manager.go

```diff
@@ -34,6 +34,7 @@ import (
 	v1qos "k8s.io/kubernetes/pkg/apis/core/v1/helper/qos"
 	"k8s.io/kubernetes/pkg/features"
 	statsapi "k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1"
+	"k8s.io/kubernetes/pkg/kubelet/eviction/api"
 	evictionapi "k8s.io/kubernetes/pkg/kubelet/eviction/api"
```

Comment:
I think this is duplicating the import? Unless I'm misreading something? Could we use `evictionapi`?